### DISCUSSION TITLE
how would you export the playlist.db to a simple text file of all urls?

### DISCUSSION THREAD

**Discussion by tracure1337** [FreeTubeApp/FreeTube#2412] [Q&A] [answered]
https://github.com/FreeTubeApp/FreeTube/blob/950a9f728ecb2aefae9bbfcba4b46ce71d114298/src/datastores/index.js

I thought about migrating the data to mongodb but there must surely be an easier way?

another way is to reverse this and export to sqlite.

https://github.com/Mido22/sqlite-to-nedb/blob/master/app.js

Or can I somehow just grep it?

**Comment by GilgusMaximus** [comment, marked as answer]
The database follows a basic format:
{"playlistName":"Favorites","videos":[{video1}, {video2}]} and each video has some properties as "key": "value" pairs separated by commas. So you should easily be able to grep these. You can just open the files with any text editor
